CollAction/Controllers/ProjectsController.cs

```diff
@@ -67,8 +67,12 @@ public async Task<IActionResult> Details(int? id)
         [Authorize]
         public async Task<IActionResult> Create()
         {
+            var user = await  _userManager.GetUserAsync(User);
+
             return View(new CreateProjectViewModel
             {
+                ProjectStarterFirstName = user?.FirstName,
```

Comment:
True. Replaced the null-coalescion with a check on the user (since it can return null according to the docs, if the user has no UserIdClaimType). But that would be an invalid operation. Throwing exception instead.